src/lib/pages/backoffice/Item/ItemForms/ItemEditor/ItemForm/ItemForm.js

```diff
@@ -92,15 +92,36 @@ export class ItemForm extends Component {
     goTo(BackOfficeRoutes.itemDetailsFor(getIn(response, 'data.metadata.pid')));
   };
 
+  shouldModalDisplay(initialValues, values) {
+    if (initialValues.document.pid !== values.document.pid) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
   render() {
     const {
       data: { metadata },
       data,
       successSubmitMessage,
       title,
       pid,
+      itemPastLoans,
     } = this.props;
     const initialValues = data ? this.prepareData(metadata) : {};
+    const itemOnLoan =
+      data && data.metadata && data.metadata.circulation
+        ? invenioConfig.CIRCULATION.loanActiveStates.includes(
+            data.metadata.circulation.state
+          )
+        : false;
+    const modal = {
+      header: 'Possible inconsistency',
```

Comment:
Just wondering: what about if we keep it simple and we disable the document selector field also when `itemPastLoans > 0`? Then we add a similar message on the select to explain why it cannot be changed.

I am worried that the will anyway proceed, and then it can really create inconsistency around :)
This will also simply the form and avoid the modal.

Reply:
Absolutley, if that's an option would simplify things a lot. I didn't consider this option because I thought that the change of the document field in the item form is a "requirement", if it's not, I would go for this option.

Reply:
I agree, lets block changing the item's document completely if there are loans